app/Exceptions/Handler.php

```diff
@@ -40,6 +58,18 @@ class Handler extends ExceptionHandler
      */
     public function report(Exception $exception)
     {
+        if ($exception instanceof TokenMismatchException) {
+            $logData = [
+                'requestToken' => request()->header('x-csrf-token'),
+                'sessionToken' => session()->token(),
+                'session' => session()->all(),
+                'user' => request()->user(),
+                'requestUrl' => request()->url()
+            ];
+            $message = '419 CSRF Token Mismatch. ' . collect($logData)->toJson();
+            Log::debug($message);
```

Comment:
Will `Log::debug()` run outside of a local environment?

Reply:
Just took a gander at the source, doesn't look like it checks for any sort of levels for different environments, so you are indeed correct